js/popup.js

```diff
@@ -1,16 +1,41 @@
 const {storage} = chrome;
 
+const currencyMap = {
+  "GBP": "£",
+  "USD": "$",
+  "EUR": "€",
+  "RUB": "₽",
+  "BTC": "Ƀ"
+};
+
 const form = document.querySelector(".js-currencies"),
   feed = document.querySelector(".js-feed"),
+  wallet = document.querySelector('.js-wallet'),
   buttons = document.querySelector(".buttons");
 
 const updateLabel = name => (name === "BTC" ? "mBTC" : name);
 
-const renderCurrencies = ({data, select, feed}) => {
+const renderWallet = (wallet, balance, data, select) => {
+  const safeSelect = select || "USD";
+  document.querySelector(".js-wallet").value = wallet || "";
+  if(balance) {
+    let currencyDecimalPoints = 2;
+    if(select === "BTC") {
+      currencyDecimalPoints = 8;
+    }
+    const calculatedBalance = Number(Number(balance)/100000000),
+          calculatedCurrencyBalance = Number(calculatedBalance * Number(data[safeSelect])).toFixed(currencyDecimalPoints);
```

Comment:
Please use something like this to format numbers:
`
const numFormat = (num, format) => new Intl.NumberFormat(
        navigator.language,
        { maximumFractionDigits: format },
    ).format(num);
`